invenio_access/alembic/2f63be7b7572_create_access_tables.py

```diff
@@ -61,7 +61,7 @@ def upgrade():
         sa.Column('exclude', sa.Boolean(name='exclude'), server_default='0',
                   nullable=False),
         sa.Column('argument', sa.String(length=255), nullable=True),
-        sa.Column('user_id', sa.Integer(), nullable=True),
+        sa.Column('user_id', sa.Integer(), nullable=False),
```

Comment:
only put it to "NOT NULL" so we have a predictable behavior. If set to NULL, we can have some issues, as described in the decsription